modules/page/html/render/polls.js

```diff
@@ -0,0 +1,78 @@
+var { h, Value, computed, when } = require('mutant')
+
+// TODO: I'd rather this came from scuttle-poll but inject means I get an obs called isPoll and it's a bit meh
+var {isPoll, isPosition} = require('ssb-poll-schema')
+var nest = require('depnest')
+var ScuttlePoll = require('scuttle-poll')
+
+exports.needs = nest({
+  'feed.pull.type': 'first',
+  'feed.html.rollup': 'first',
+  // 'feed.pull.public': 'first',
+  'poll.sheet.edit': 'first',
+  'keys.sync.id': 'first',
+  // 'contact.obs.following': 'first',
+  'sbot.pull.stream': 'first',
+  'sbot.obs.connection': 'first',
+  'intl.sync.i18n': 'first'
+})
+
+exports.gives = nest('page.html.render')
+
+const OPEN = 'open'
+const CLOSED = 'closed'
+const ALL = 'all'
+
+exports.create = function (api) {
+  const i18n = api.intl.sync.i18n
+
+  return nest('page.html.render', function channel (path) {
+    if (path !== '/polls') return
+
+    var scuttlePoll = ScuttlePoll(api.sbot.obs.connection)
+    var id = api.keys.sync.id()
+
+    var mode = Value(OPEN)
```

Comment:
this is state which leads to page being changed but route note being changed.

matt wants this to use routes to do the changes instead .. i.e. those filters should go to : 
- /polls  (same as /polls/open ?)
- /polls/open
- /polls/closed
- /polls/all